Unix/buildtool

```diff
@@ -224,36 +225,50 @@ if [ -z "$target" ]; then
             os=SUNOS
             arch=SPARC
             compiler=SUNPRO
+            distro=`uname -s`
+            distro_version=`uname -v`
             ;;
         i86pc:SunOS:*:*)
             os=SUNOS
             arch=I86PC
             compiler=SUNPRO
+            distro=`uname -s`
+            distro_version=`uname -v`
             ;;
         *:AIX:*:*)
             os=AIX
             arch=PPC
             compiler=IBM
+            distro=`uname -s`
+            distro_version=`uname -v`
             ;;
         ia64:HP-UX:*:*)
             os=HPUX
             arch=IA64
             compiler=HP
+            distro=`uname -s`
+            distro_version=`uname -v`
             ;;
         9000/800:HP-UX:*:*)
             os=HPUX
             arch=PARISC
             compiler=HP
+            distro=`uname -s`
+            distro_version=`uname -v`
             ;;
         i386:Darwin:*:Darwin* | x86_64:Darwin:*:Darwin*)
             os=DARWIN
             arch=IX86
             compiler=GNU
+            distro=`uname -s`
+            distro_version=`uname -v`
             ;;
         ppc*:Linux:*)
             os=LINUX
             arch=PPC
             compiler=GNU
+            distro=`lsb_release -i | awk -F":" '{ print $2 }'`
+            distro_version=`lsb_release -r | awk -F":" '{ print $2 }'`
```

Comment:
Note that this will give you leading spaces, which you may or may not care about:

```
jeffcof:code> lsb_release -r | awk -F":" '{ print $2 }'
        7.3.1611
jeffcof:code>
```